### ISSUE TITLE
NPM package has console.log statement inside readFile() function, outputting the CSS at root

### ISSUE THREAD

**Issue by gremo** [eriklharper/postcss-nested-import#5]
As per title, you should publish a new NPM version as the published version as a `console.log` statement. This cause the imported CSS outputted twice (at root and where the `@import` is actually used).

I'll paste the entire content:

```js
const postcss = require('postcss');
const fs = require('fs');

/**
 * ParseImportPath
 * @param  {[type]} path [description]
 * @return {[type]}      [description]
 */
function parseImportPath(path) {
  const matches = path.trim().match(/^['"](.+?)['"](.*)/);
  const modes = matches[2].trim().split(" ");
  const fragment = matches[1];
  return fragment;
}

/**
 * ReadFile
 * @param  {[type]} file [description]
 * @return {[type]}      [description]
 */
function readFile(file) {
  return new Promise((resolve, reject) => {
    fs.readFile(file, 'utf-8', (err, contents) => {
      if (err) {
        return reject(err);
      }
      console.log(contents);
      resolve(contents);
    });
  });
}

/**
 * PostCSS Nested Import Plugin
 * @type {[type]}
 */
module.exports = postcss.plugin('postcss-nested-import', () => {
  return (root, result) => {
    return new Promise((resolve, reject) => {
      let prom = Promise.resolve();
      root.walkAtRules('import', importAtRule => {
        if (importAtRule.params) {
          const path = parseImportPath(importAtRule.params);
          if (path == null) {
            return;
          }
          prom = prom.then(() => {
            return readFile(path).then(fileContents => {
              // console.log(importAtRule);
              // console.log(fileContents);
              return importAtRule.replaceWith(fileContents);
            });
          });
        }
      });
      resolve(prom);
    });
  };
});

```

**Comment by revelt** [comment]
hi Tom @colourgarden, sorry about that, we're working on it. I've got push rights to GH but not npm, I pinged Erik, we'll sort this.

**Comment by revelt** [comment]
we've just released 0.2.0, please upgrade
